extensions/sql-migration/src/wizard/targetSelectionPage.ts

```diff
@@ -72,12 +71,26 @@ export class TargetSelectionPage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS, 'margin': '0' }
 			}).component();
 
+		this._certMigrationRequiredInfoBox = this._view.modelBuilder.infoBox()
+			.withProps({
+				text: constants.TDE_WIZARD_CERT_MIGRATION_BUTTON_MUST_BE_CLICKED,
+				style: 'information',
```

Comment:
if this is a blocker, perhaps change the style to warning or error instead of informational?

Reply:
I'm on the fence about this.  There's no way for the user to not click the "Migrate certificates" button, so what are we warning about?

That said it seems like you, @AkshayMata and @brian-harris all prefer the warning.  I suppose it can't hurt.